tests/integ/conftest.py

```diff
@@ -220,6 +220,14 @@ def session(db_parameters, resources_path, sql_simplifier_enabled, local_testing
         set_up_external_access_integration_resources(
             session, rule1, rule2, key1, key2, integration1, integration2
         )
+        # this temporarily fix the BCR change introduced in the bundle 2024-03
+        # check SNOW-1231022 for more context and follow-up action
+        session.sql(
+            "alter session set INCLUDE_DYNAMIC_TABLES_WITH_TABLE_KIND=true"
```

Comment:
Do we have the right role to run this alter session command (if this is not a customer-facing param)?